reliquery/relic.py

```diff
@@ -264,6 +266,32 @@ def get_image(self, name: str) -> BytesIO:
     def list_images(self) -> List[str]:
         return self.storage.list_keys([self.relic_type, self.name, "images"])
 
+    def add_json(self, name: str, jsonData: dict) -> None:
```

Comment:
I would normalize the parameter and return value typing to use the same as previous code for now. 
dict -> Dict and such.

Reply:
Also use the lowercase with underscores instead of camelCase